website/source/docs/builders/amazon-ebs-volume.html.md

```diff
@@ -0,0 +1,331 @@
+         This is most useful for selecting a daily distro build.
+
+-   `spot_price` (string) - The maximum hourly price to pay for a spot instance
+    to create the AMI. Spot instances are a type of instance that EC2 starts
+    when the current spot price is less than the maximum price you specify. Spot
+    price will be updated based on available spot instance capacity and current
+    spot instance requests. It may save you some costs. You can set this to
+    "auto" for Packer to automatically discover the best spot price or to "0"
+    to use an on demand instance (default).
+
+-   `spot_price_auto_product` (string) - Required if `spot_price` is set
+    to "auto". This tells Packer what sort of AMI you're launching to find the
+    best spot price. This must be one of: `Linux/UNIX`, `SUSE Linux`, `Windows`,
+    `Linux/UNIX (Amazon VPC)`, `SUSE Linux (Amazon VPC)`, `Windows (Amazon VPC)`
+
+-   `ssh_keypair_name` (string) - If specified, this is the key that will be
+    used for SSH with the machine. By default, this is blank, and Packer will
+    generate a temporary keypair unless
+    [`ssh_password`](/docs/templates/communicator.html#ssh_password) is used.
+    [`ssh_private_key_file`](/docs/templates/communicator.html#ssh_private_key_file)
+    must be specified with this.
+
+-   `ssh_private_ip` (boolean) - If true, then SSH will always use the private
+    IP if available.
+
+-   `subnet_id` (string) - If using VPC, the ID of the subnet, such as
+    "subnet-12345def", where Packer will launch the EC2 instance. This field is
+    required if you are using an non-default VPC.
+
+-   `temporary_key_pair_name` (string) - The name of the temporary keypair
+    to generate. By default, Packer generates a name with a UUID.
+
+-   `token` (string) - The access token to use. This is different from the
+    access key and secret key. If you're not sure what this is, then you
+    probably don't need it. This will also be read from the `AWS_SESSION_TOKEN`
+    environmental variable.
+
+-   `user_data` (string) - User data to apply when launching the instance. Note
+    that you need to be careful about escaping characters due to the templates
+    being JSON. It is often more convenient to use `user_data_file`, instead.
+
+-   `user_data_file` (string) - Path to a file that will be used for the user
+    data when launching the instance.
+
+-   `vpc_id` (string) - If launching into a VPC subnet, Packer needs the VPC ID
+    in order to create a temporary security group within the VPC. Requires `subnet_id`
+    to be set.
+
+-   `windows_password_timeout` (string) - The timeout for waiting for a Windows
+    password for Windows instances. Defaults to 20 minutes. Example value: "10m"
+
+-   `shutdown_behaviour` (string) - Automatically terminate instances on shutdown
+    incase packer exits ungracefully. Possible values are "stop" and "terminate",
+    default is stop.
+
+## Basic Example
```

Comment:
I think this is a way too big example to serve as a "basic example". I think it should be something like:

```json
{
   "type" : "amazon-ebsinit",
   "secret_key" : "YOUR SECRET KEY HERE",
   "access_key" : "YOUR KEY HERE",
   "region" : "us-east-1",
   "ssh_username" : "ubuntu",
   "instance_type" : "t2.medium",
   "source_ami" : "ami-40d28157",
   "ebs_volumes" : [
      {
         "volume_type" : "gp2",
         "device_name" : "/dev/xvdf",
         "delete_on_termination" : false,
         "tags" : {
            "zpool" : "data",
            "Name" : "Data1"
         },
         "volume_size" : 10
      },
      {
         "volume_type" : "gp2",
         "device_name" : "/dev/xvdg",
         "tags" : {
            "zpool" : "data",
            "Name" : "Data2"
         },
         "delete_on_termination" : false,
         "volume_size" : 10
      },
      {
         "volume_size" : 10,
         "tags" : {
            "Name" : "Data3",
            "zpool" : "data"
         },
         "delete_on_termination" : false,
         "device_name" : "/dev/xvdh",
         "volume_type" : "gp2"
      }
   ]
}
```